src/components/nav-tabs/nav-tabs.tsx

```diff
@@ -1,13 +1,29 @@
 import { CITIES } from '../../const';
+import { useAppDispatch } from '../../hooks/store';
+import { TCity } from '../offer-card/types';
+import { setCity } from '../../store/action';
+
+type NavTabsProps = {
+  currentCity: TCity;
+}
+
+function NavTabs({currentCity}: NavTabsProps): JSX.Element {
+  const dispatch = useAppDispatch();
 
-function NavTabs(): JSX.Element {
   return (
     <div className="tabs">
       <section className="locations container">
         <ul className="locations__list tabs__list">
           {CITIES.map((city) => (
             <li className="locations__item" key={city}>
-              <a className="locations__item-link tabs__item" href="#">
+              <a
+                className={`locations__item-link tabs__item ${currentCity.name !== city || 'tabs__item--active'}`}
```

Comment:
Через classnames/clsx + отрицание всегда увеличивает когнитивную нагрузку есть даже правило для еслинта которое чекает это no-negated-condition
```suggestion
                className={clsx('locations__item-link tabs__item', currentCity.name === city && 'tabs__item--active)}
```